test/ErrorHandler-test.js

```diff
@@ -1,14 +1,16 @@
 import { describe, it } from 'mocha'
+import '../src/Promise'
```

Comment:
> does silenceError actually need to use an Action and call _runAction to silence the error?

Not really.  My original intent was for silenceError to be safely callable on any promise, which is why it works the way it does (e.g. you can call it safely on a pending promise that eventually rejects).  It's not exported in the public API, tho, so we can change how it works internally if that will simplify things.